Nodejs/Product/Npm/SPI/NpmController.cs

```diff
@@ -100,18 +95,19 @@ internal class NpmController : AbstractNpmLogSource, INpmController {
         }
 
         public void Refresh() {
-            RefreshAsync().ContinueWith(t => {
-                var ex = t.Exception;
+            try {
+                RefreshImplementation();
+            } catch (Exception ex) {
                 if (ex != null) {
                     OnOutputLogged(ex.ToString());
 #if DEBUG
                     Debug.Fail(ex.ToString());
 #endif
                 }
-            });
+            }
         }
 
-        public async Task RefreshAsync() {
+        private void RefreshImplementation() {
```

Comment:
Ahh, gotcha - yeah, nevermind. Now that I take a closer look, let's leave it as is... the behavior should be exactly the same because an async method that returns void and doesn't have await operators would run synchronously anyways.